patch/patcher.go

```diff
@@ -178,7 +194,15 @@ func (p *Patcher) scanField(f *protogen.Field) {
 	if newName == "" && o != nil && (p.isRenamed(m.GoIdent) || p.isRenamed(o.GoIdent)) {
 		// Implicitly rename this oneof field because its parent(s) were renamed.
 		newName = f.GoName
+	} else {
+		if len(p.fileInitialisms) > 0 {
```

Comment:
Thinking this should run on all names, even if renamed, so it can fix nested oneof names.